### PR TITLE
OCPBUGS-16192: Use no uplink br-ex in mutlinode mode

### PR DESCRIPTION
**Pull request by pliurh**
1. Enable no uplink mode for ovnkube.
2. Configure dummy IP for br-ex interface.

### CODE REVIEW

File: scripts/multinode/configure-pri.sh [hunk 1]
```diff
@@ -43,6 +43,9 @@ function configure_microshift() {
     sleep 5
     sudo systemctl start microshift-ovs-init.service
 
+    # Add dummy IP to br-ex
+    sudo ip addr add 10.44.0.0/32 dev br-ex
```
Comment (ggiguash) [inline scripts/multinode/configure-pri.sh:47]: Is this call idempotent? In other words, if we run the script on the same machine twice, does it work for the second time without failing?

File: scripts/multinode/configure-sec.sh [hunk 2]
```diff
@@ -40,6 +40,9 @@ function configure_microshift() {
     sleep 5
     sudo systemctl start microshift-ovs-init.service
 
+    # Add dummy IP to br-ex
+    sudo ip addr add 10.44.0.0/32 dev br-ex
```
Comment (ggiguash) [inline scripts/multinode/configure-sec.sh:44]: Why are we choosing the IP from 10.44.0.0 subnet? Is it related to some default configuration setting?

File: scripts/multinode/configure-pri.sh (hunk 1 above)
Comment (ggiguash) [reply scripts/multinode/configure-pri.sh:47]: Also, can we please add a comment in the script explaining why this is done?

File: scripts/multinode/configure-sec.sh (hunk 2 above)
Comment (pliurh) [reply scripts/multinode/configure-sec.sh:44]: We use this IP in the single node cluster as the default KAS advertise address. We can use any IP as long as it doesn't overlap with any other internal/external IP ranges.

File: scripts/multinode/configure-pri.sh (hunk 1 above)
Comment (pliurh) [reply scripts/multinode/configure-pri.sh]: fixed. PTAL

File: scripts/multinode/configure-pri.sh [hunk 3]
```diff
@@ -43,6 +43,9 @@ function configure_microshift() {
     sleep 5
     sudo systemctl start microshift-ovs-init.service
 
+    # OVN-K expects br-ex to have IP address assigned, add dummy IP to br-ex.
+    if ! ip addr show br-ex | grep '10.44.0.0/32'; then sudo ip addr add 10.44.0.0/32 dev br-ex; fi
```
Comment (ggiguash) [inline scripts/multinode/configure-pri.sh:47]:
```suggestion
    if ! ip addr show br-ex 2>/dev/null | grep -q '10.44.0.0/32'; then
        sudo ip addr add 10.44.0.0/32 dev br-ex
    fi
```

File: scripts/multinode/configure-sec.sh [hunk 4]
```diff
@@ -40,6 +40,9 @@ function configure_microshift() {
     sleep 5
     sudo systemctl start microshift-ovs-init.service
 
+    # OVN-K expects br-ex to have IP address assigned, add dummy IP to br-ex.
+    if ! ip addr show br-ex | grep '10.44.0.0/32'; then sudo ip addr add 10.44.0.0/32 dev br-ex; fi
```
Comment (ggiguash) [inline scripts/multinode/configure-sec.sh:44]:
```suggestion
    if ! ip addr show br-ex 2>/dev/null | grep -q '10.44.0.0/32'; then
        sudo ip addr add 10.44.0.0/32 dev br-ex
    fi
```

**Pull request merged**